src/css/profile/mobile/common/card.less

```diff
@@ -20,6 +20,7 @@
 		.ui-content-subheader {
 			display: flex;
 			align-items: center;
+			white-space: nowrap;
```

Comment:
shouldn't we have 
 text-overflow: ellipsis in such case?

Reply:
This is more complicated. We cannot use `text-overflow` without `overflow: hidden` but `overflow: hidden` stay in opposite to `display: flex` and doesn't. So, we should wrap text in `span` element and define `max-width` for its. In this place would be nice use marquee widget but it all is out of scope guideline.

Reply:
If this change is according to the guidelines I think this is OK, I was just afraid that text will be cut unexpectedly :)

Reply:
Thank you for review :)